subcommands/targets/show.go

```diff
@@ -45,19 +54,51 @@ func init() {
 	showAppCmd.Flags().Bool("manifest", false, "Show an app docker manifest")
 
 	sbomCmd := &cobra.Command{
-		Use:   "sboms <version> [<build/run>]",
+		Use:   "sboms <version> [<build/run> [<artifact>]] ",
 		Short: "Show SBOMs for a specific target.",
 		Run:   doShowSboms,
-		Args:  cobra.RangeArgs(1, 2),
+		Args:  cobra.RangeArgs(1, 3),
 		Example: `
   # Show all SBOM files for Target version 42:
   fioctl targets show sboms 42
 
   # Show a subset of the SBOMS for this target. In this case, the 32-bit Arm
   # container SBOMS:
-  fioctl targets show sboms 42 41/build-armhf`,
+  fioctl targets show sboms 42 41/build-armhf
+ 
+  # Show overview of a specific SBOM:
+  fioctl targets show sboms 42 41/build-armhf alpine:latest/arm.spdx.json
+  
+  # Show overview of a specific SBOM as CSV:
+  fioctl targets show sboms --format csv 42 41/build-armhf alpine:latest/arm.spdx.json`,
+	}
+
+	sbomFormats = append(sbomFormats, sbomFormat{
+		name:        "table",
+		contentType: "application/spdx.json",
+	})
+	sbomFormats = append(sbomFormats, sbomFormat{
+		name:        "spdx",
+		contentType: "application/spdx.json",
+	})
+	sbomFormats = append(sbomFormats, sbomFormat{
+		name:        "cyclonedx",
+		contentType: "application/cyclone.json",
+	})
+	sbomFormats = append(sbomFormats, sbomFormat{
+		name:        "csv",
+		contentType: "text/csv",
+	})
+
+	allowed := ""
+	for i, fmt := range sbomFormats {
+		if i > 0 {
```

Comment:
I'd recommend to check for `f.name != "table"` here explicitly.

Reply:
how about even simpler and not use any kind of "needle"?
```
        allowed := ""
	for _, fmt := range sbomFormats {
		if len(allowed) > 0 {
			allowed += ","
		}
		allowed += fmt.name
	}
```
doesn't see like we should make comma joining logic be dependent on the order of the inputs?